src/core/server/rendering/rendering_service.tsx

```diff
@@ -338,12 +341,14 @@ export class RenderingService {
    * @returns {boolean} indicate if user input title is valid/invalid
    */
   public isTitleValid = (title: string, configName?: string): boolean => {
-    if (!title || title.length > 36) {
+    if (!title) {
+      return false;
```

Comment:
So we don't log any thing here?

Reply:
I don't think it's necessary to log something here because we just use the default title in this case.

Reply:
Oh, I see now.  It prevents logging here on purpose.